docs/build_docs.py

```diff
@@ -21,15 +22,16 @@ def get_abs_path(path, root):
 
 
 if __name__ == '__main__':
-
     args = parser.parse_args()
 
     docs_dir = pathlib.Path(__file__).parent.absolute()
     work_dir = get_abs_path(path=args.work_dir, root=docs_dir)
     prefix = get_abs_path(path=args.prefix, root=docs_dir)
 
     # Build the docs with sphinx-build
-    status = subprocess.check_call(
-        ['sphinx-build', '-v', '-d', work_dir, docs_dir, prefix],
-        stderr=subprocess.STDOUT,
-        shell=sys.platform == "win32")
+    subprocess.check_call([
+        'sphinx-build', '-v', '-b', args.builder, '-d', work_dir, docs_dir,
+        prefix
+    ],
+                          stderr=subprocess.STDOUT,
+                          shell=sys.platform == "win32")
```

Comment:
Formatting?

Reply:
This was produced by yapf. I tried to persuade it into a different format but it was stubborn...